### ISSUE TITLE
Add support for Node 12

### ISSUE THREAD

**Issue by smoliakov** [rafinskipg/git-changelog#86]
### Using:
- Node 12.8.0
- OS Windows 10
- git-changelog 1.1.2

### Problem:
```
$ ./node_modules/.bin/git-changelog
fs.js:27
const { Math, Object } = primordials;
                         ^

ReferenceError: primordials is not defined
    at fs.js:27:26
    at req_ (F:\Web\frontend\node_modules\natives\index.js:143:24)
    at Object.req [as require] (F:\Web\frontend\node_modules\natives\index.js:55:10)
    at Object.<anonymous> (F:\Web\frontend\node_modules\git-changelog\node_modules\graceful-fs\fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:868:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:879:10)
    at Module.load (internal/modules/cjs/loader.js:731:32)
    at Function.Module._load (internal/modules/cjs/loader.js:644:12)
    at Module.require (internal/modules/cjs/loader.js:771:19)
    at require (internal/modules/cjs/helpers.js:68:18)
```

### Notes:
`primordials` is a native Node.js core part and there were changes in 11+ regarding this.

**Comment by Phylu** [comment]
@rafinskipg I assume, that this repository is not active anymore and this will not be resolved?

**Comment by Phylu** [comment]
Thanks for the fix! Is it possible to create a new release with this fix included?

**Comment by glebcha** [comment]
> Thanks for the fix! Is it possible to create a new release with this fix included?

I hope @rafinskipg will publish new version soon, but for now you can add git repository as dependency.

**Comment by glebcha** [comment]
@smoliakov @Phylu new version published, please check 1.2.0
